### PR TITLE
MAISTRA-2399 Move doc generation from operator to api repo

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,12 +7,17 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/google/go-cmp v0.5.2
 	github.com/maistra/xns-informer v0.0.0-20210707160032-977ec17e2e0e
+	github.com/pkg/errors v0.9.1
+	github.com/spf13/pflag v1.0.5
 	google.golang.org/grpc v1.28.1
 	k8s.io/api v0.20.1
+	k8s.io/apiextensions-apiserver v0.18.6
```
Comment (jwendell) [inline go.mod]: Should this follow the others, i.e., be 0.20.1?

Comment (luksa) [reply go.mod]: Yes. Thanks.

Comment (luksa) [reply go.mod]: Hmm, looks like we pin this to 0.18.3 in istio-operator. Not sure why.

Comment (jwendell) [reply go.mod]: That was when we tried to keep operator in line with istio and api. Now that api and istio are on 0.20, we are free to bump operator too. Not only this module, but all k8s.io/* that are currently 0.18 or 0.19.

**Review by jwendell — APPROVED** [review]

**Pull request merged**